optimize/client/src/components/Footer/Footer.tsx

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-
-import {t} from 'translation';
-import {useUiConfig} from 'hooks';
-
-import './Footer.scss';
-
-export default function Footer() {
-  const {optimizeVersion} = useUiConfig();
-
-  const timezoneInfo =
-    t('footer.timezone') + ' ' + Intl.DateTimeFormat().resolvedOptions().timeZone;
-
-  return (
-    <footer className="Footer">
-      <div title={timezoneInfo} className="timezone">
-        {timezoneInfo}
-      </div>
```

Comment:
The timezone information is an important thing in Optimize and should not be removed.
I think it should be moved to the user menu as well. We can try adding a seperator and putting it below the copyright info.

Reply:
I cant see any place where we can pass this. C3 navigation has defined props and its not really allowing to pass anything more then those props. There is `bottomElements` section, but it renders buttons and takes props to use with buttons.

Reply:
I will go for it and use that `bottomElements` for this, but it is really hacky

Reply:
@misiekhardcore can we try the suggested solution in the slack thread above?